marklogic-client-api/src/main/java/com/marklogic/client/impl/DatabaseClientPropertySource.java

```diff
@@ -159,7 +161,11 @@ private DatabaseClientFactory.SecurityContext newSecurityContext() {
 		final String authType = determineAuthType(connectionString);
 
 		final SSLUtil.SSLInputs sslInputs = buildSSLInputs(authType);
-		DatabaseClientFactory.SecurityContext securityContext = newSecurityContext(authType, connectionString, sslInputs);
+		String sslProtocol = getSSLProtocol(authType);
+		if ("default".equalsIgnoreCase(sslProtocol) || sslProtocol == null) {
```

Comment:
You don't need this here, as it's only for cert auth and the cert auth will handle the default value. 

In fact, to make this more clear, rename the sslProtocol variable to `sslProcotolForCertificateAuth`. That's making it clear that cert auth is the oddball here that can't be happy with just an SSLContext; it needs an SSL protocol too so it can build its own SSLContext based on a cert file/password.